CoffeeChat/Controller/SchedulingTimeViewController.swift

```diff
@@ -47,39 +49,69 @@ class SchedulingTimeViewController: UIViewController {
     private var timeSections: [Section] = []
 
     // MARK: - Data
-    private let finishButtonSize = CGSize(width: 175, height: 50)
+    private let nextButtonSize = CGSize(width: 175, height: 50)
     private let interitemSpacing: CGFloat = 4
     private let sectionInsets = UIEdgeInsets(top: 0, left: 8, bottom: 0, right: 8)
 
     private let dayCellReuseId = "dayCellReuseIdentifier"
     private let timeCellReuseId = "timeCellReuseIdentifier"
 
+    private var allAfternoonTimes = ["1:00", "1:30", "2:00", "2:30", "3:00", "3:30", "4:00", "4:30"]
```

Comment:
I feel like I get confused when I look at all these variables that are like really similar but all do slightly different things. maybe some comments may be useful here? or later when you set like `afternoonTimes = allAfternoonTimes`